org.eclipse.jgit.test/tst/org/eclipse/jgit/api/CommitCommandTest.java

```diff
@@ -532,6 +534,38 @@ public void commitAmendWithAuthorShouldUseIt() throws Exception {
 		}
 	}
 
+	@Test
```

Comment:
@guw modifying this test based on latest changes would suffice, right?

I can try running 'git verify-commit' on this commit hash as well